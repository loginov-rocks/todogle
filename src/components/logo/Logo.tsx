import { Typography } from '@material-ui/core';
import * as React from 'react';

const Logo = () => (
  <Typography component="h1" variant="h4">Todogle</Typography>
);

export default Logo;
