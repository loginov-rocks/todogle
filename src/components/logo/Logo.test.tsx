import { shallow } from 'enzyme';
import * as React from 'react';

import Logo from './Logo';

it('matches snapshot', () => {
  const wrapper = shallow(
    <Logo />,
  );

  expect(wrapper).toMatchSnapshot();
});
