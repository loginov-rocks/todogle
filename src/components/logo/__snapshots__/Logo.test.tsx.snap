// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<WithStyles(ForwardRef(Typography))
  component="h1"
  variant="h4"
>
  Todogle
</WithStyles(ForwardRef(Typography))>
`;
